src/middleware/persist.ts

```diff
@@ -271,6 +271,8 @@ const oldImpl: PersistImpl = (config, baseOptions) => (set, get, api) => {
     api,
   )
 
+  api.getServerState = () => configResult
+
```

Comment:
not sure if this is also needed in `oldImpl` ?

Reply:
We don't modify oldImpl. Just changing newImpl is fine.

Reply:
reverted here: 9c7b13e3